generators/aggregatedGenerator.js

```diff
@@ -0,0 +1,128 @@
+const { metricsList } = require('./getMetricFunctions')
+
+const CURRENCY_DESCRIPTION = `
+* @param {string} currency The metric is calculated using a currency of choice.
+* Available currencies: (currencies)`
+
+const TIMEBOUND_DESCRIPTION = `
+* @param {string} timeBound The metric is calculated only by taking into account the
+* tokens/coins that have moved in the past number of years or days.`
+
+const BALANCES_DESCRIPTION = `
+* @param {string} balance Interval of amount of tokens. Example: "0-0.001"`
+
+const AGGREGATION_DESCRIPTION = `
+* @param {string} aggregation Aggregation for the timeseries metrics. Example: "LAST"
+`
+
+const FORBIDDEN_METRICS = [ // Aggregated not implemented for these metrics
```

Comment:
Isn't IGNORED_METRICS a better name?

Reply:
Yeah, it would be, I just thought of that name on the go :D ignored is sure better, will change it